lib/chef-dk/command/env.rb

```diff
@@ -34,6 +34,10 @@ def initialize(*args)
         @ui = UI.new
       end
 
+      def check_license_acceptance
+        # Do not need to accept the license for checking the environment
```

Comment:
We do not request the license for `knife` which made me think we shouldn't request the license here. But then I remembered that `knife` is not a trademarked term so that is why we don't request the EULA there, so I'm going to request the EULA for all `chef` subcommands besides `-v` and `-h`